src/web/mjs/connectors/wuqimh.mjs

```diff
@@ -0,0 +1,26 @@
+import SinMH from './templates/SinMH.mjs';
+
+export default class wuqimh extends SinMH {
+
+    constructor() {
+        super();
+        super.id = 'wuqimh';
+        super.label = 'wuqimh';
+        this.tags = [ 'manga','hentai','chinese' ];
+        this.url = 'http://www.wuqimh.com';
+        this.requestOptions.headers.set('x-referer', this.url);
+
+        this.pathMatch = '/list/p-(\\d+)';
+        this.queryMangasPageCount = 'div.pager-cont span.pager a:nth-last-of-type(2)';
+        this.queryChapters = 'div.chapter-list ul li a';        
+
+        this.scriptPages =`
+            new Promise((resolve, reject) => {
```

Comment:
Can be simplified to
`new Promise(resolve => resolve(cInfo.fs.map(img => 'http://' + pageConfig.host.auto + img)));`

Reply:
ok but how did u get to that answer so fast

Reply:
nvm ik how you got to it
im just dumb